app/views/application/equality-monitoring/free-school-meals.html

```diff
@@ -1,31 +1,52 @@
-{% extends "_form.html" %}
+{% extends "layout.html" %}
 {% set parent = "Equality and diversity questions" %}
-{% set title = "If you finished school after 1980, were you eligible for free school meals at any point during your school years?" %}
+{% set title = "Did you ever get free school meals in the UK?" %}
 
 {% block pageNavigation %}
   {{ govukBackLink({
     href: paths.back
   }) }}
 {% endblock %}
 
-{% block primary %}
-  {{ govukRadios({
-    items: [{
-      text: "Yes"
-    }, {
-      text: "No"
-    }, {
-      text: "Not applicable, I finished school before 1980 or went to school overseas"
-    }, {
-      text: "I don't know"
-    }, {
-      divider: "or"
-    }, {
-      text: "Prefer not to say"
-    }]
-  } | decorateApplicationAttributes(["equality-monitoring", "free-school-meals"])) }}
-
-  {{ govukButton({
-    text: "Continue"
-  }) }}
+{% block content %}
+
+  <form{% if formaction %} action="{{ formaction }}"{% endif %} method="post" novalidate>
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        <span class="govuk-caption-l">{{ parent }}</span>
+        <h1 class="govuk-heading-l">Free school meals</h1>
+
+        <p class="govuk-body">Training providers will not see your answer to this question.</p>
+
+        <p class="govuk-body">This question helps the Department for Education understand the financial background of trainee teachers. This is because free school meals have generally only been available to children from  families that receive benefits.</p>
+
+        {{ govukRadios({
+          fieldset: {
+            legend: {
+              text: title,
+              classes: "govuk-fieldset__legend--m"
+            }
+          },
+          items: [{
+            text: "Yes"
+          }, {
+            text: "No"
```

Comment:
I'd be tempted for one or both of these to be a wordier answer to avoid the problems with yes/no questions.

Reply:
We decided against that in this instance, as hopefully the question is now short and clear enough to not need it.